monetdbe/_cffi/convert/bind.py

```diff
@@ -10,8 +10,8 @@ def monetdbe_int(data: int) -> ffi.CData:
         return ffi.new("int *", data)
 
 
-def bind_str(data: str) -> bytes:
-    return str(data).encode()
+def bind_str(data: str) -> ffi.CData:
+    return ffi.new("char[]", str(data).encode())
```

Comment:
why is this required?

Reply:
Hmm I think because monetdbe expects a deep copy (so an array) of the C string and not a reference to the original. I remembered that I got corruption errors in certain situations without this change.